### DISCUSSION TITLE
Can this be used in Android Studio?

### DISCUSSION THREAD

**Discussion by otberger** [aeharding/metar-taf-parser#91] [Q&A] [answered]
Given a taf in Jetpack Compose Composable, can I use this library to decode it?

**Comment by SkySails** [comment, marked as answer]
Hello!

**TL;DR:** Based on the information you have provided so far, the answer is no.

Please provide more information on your full scenario. The project that you have currently interacted with is a Typescript project, and it does not sound like you are working with Typescript if you are talking about Jetpack Composables.

It is worth noting that this project is a port of a Python project with the same name, which is mentioned in the first sentence of the README:

> Parser for METeorological Aerodrome Reports (METARs) and Terminal Aerodrome Forecasts (TAFs). This is a port of [python-metar-taf-parser](https://github.com/mivek/python-metar-taf-parser) to Typescript with some additional features.

The creator behind the Python project has also made a Java implementation. **Based on your mention of Jetpack Compose, you are likely doing Android development in either Kotlin or Java**, so that would probably be your best bet! You can find both mentioned projects here:

* https://github.com/mivek/MetarParser
* https://github.com/mivek/python-metar-taf-parser

I hope this helps :)
